packages/transaction-controller/src/types.ts

```diff
@@ -574,6 +574,14 @@ export interface DappSuggestedGasFees {
   maxPriorityFeePerGas?: string;
 }
 
+/**
+ * Gas values set by the user.
+ */
+export interface AdvancedGasFees {
+  maxBaseFee?: string;
```

Comment:
Should these be optional? I understood if this was saved, it should have values for both, or the whole callback return `undefined`.

Reply:
Fixed